boards/witte/linum/doc/index.rst

```diff
@@ -0,0 +1,403 @@
+  - 1 SPI
+  - 1 I2C
+  - 1 One Wire
+  - 2 DACs
+  - 6 PWM Channels
+  - 10 ADCs
+
+Supported Features
+==================
+
+The Zephyr Linum board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| RTC       | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+| RNG       | on-chip    | True Random number generator        |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | usb_device                          |
++-----------+------------+-------------------------------------+
+| CAN/CANFD | on-chip    | canbus                              |
++-----------+------------+-------------------------------------+
+| LTDC      | on-chip    | LCD Interface                       |
++-----------+------------+-------------------------------------+
+| FMC       | on-chip    | memc (SDRAM)                        |
++-----------+------------+-------------------------------------+
+| SDMMC     | on-chip    | disk access                         |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/witte_technology/linum/linum_defconfig`
+
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+
+BOARD-LEDs
+----------
+
+The LINUM-STM32H753BI has 3 software controllable LEDs.
+
+  ======= =====
+  LED RGB PINS
+  ======= =====
+  LED_R   PG2
+  LED_G   PG3
+  LED_B   PB2
+  ======= =====
```

Comment:
Comment on this whole section: I wonder how this will render in generated html page.
See https://docs.zephyrproject.org/latest/boards/st/b_u585i_iot02a/doc/index.html as example.
Did you try to launch doc generation ?

At least there will be an issue in the sections hierarchy as `Default Zephyr Peripheral Mapping:` is using same level as `USART/UART` and `USART1`.

Reply:
This was taken from nuttx documentation:

https://nuttx.apache.org/docs/latest/platforms/arm/stm32h7/boards/linum-stm32h753bi/index.html#board-leds

As you see it is just a way to generate a table with corresponding pins.

Reply:
>  As you see it is just a way to generate a table with corresponding pins.

ok, great then!

Then, I think you should still take care about 

> there will be an issue in the sections hierarchy as Default Zephyr Peripheral Mapping: is using same level as USART/UART and USART1.